Makefile

```diff
@@ -157,12 +162,35 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # Generate bundle manifests and metadata, then validate generated files.
 .PHONY: bundle
 bundle: manifests
+	mv bundle/manifests/keda.clusterserviceversion.yaml bundle/manifests/keda-olm-operator.clusterserviceversion.yaml
```

Comment:
isn't there another way? Where does it get the name that it expects? Can't be it from https://github.com/kedacore/keda-olm-operator/blob/master/PROJECT ? Just guessing.